includes/Modules/Ads.php

```diff
@@ -88,6 +93,28 @@ protected function setup_assets() {
 		);
 	}
 
+	/**
+	 * Gets required Google OAuth scopes for the module.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array List of Google OAuth scopes.
+	 */
+	public function get_scopes() {
+		$adwords_scope = 'https://www.googleapis.com/auth/adword';
```

Comment:
This should be `https://www.googleapis.com/auth/adwords` (not singular `adword` as per above). The above causes an invalid scope error:

<img width="478" alt="image" src="https://github.com/google/site-kit-wp/assets/72872375/9326013c-045c-4636-a6bd-06470493aa3e">